_data/hosting.yml

```diff
@@ -173,6 +173,13 @@ websites:
       twitter: inmotionhosting
       img: inmotion.png
       tfa: No
+      
+     - name: IO Zoom
```

Comment:
In `yaml` indentation/spaces are important. You indented this line a bit too much.